qiskit_nature/second_q/mappers/qubit_converter.py

```diff
@@ -26,6 +26,7 @@
 from qiskit.opflow.primitive_ops import Z2Symmetries
 
 from qiskit_nature import QiskitNatureError, settings
+from qiskit_nature.deprecation import warn_deprecated, DeprecatedType
```

Comment:
As this whole class is deprecated can we note it in the docs i.e this line below (I cannot comment there - had to pick a changed line)

>"""A converter from Second-Quantized to Qubit Operators.

add DEPRECATED so it shows up in the API ref. Otherwise there is no indication there at all its deprecated.

> """DEPRECATED A converter from Second-Quantized to Qubit Operators.